src/app/showcase/components/message-popup/showcase-message-popup.component.html

```diff
@@ -11,6 +11,8 @@
     <button type="button" class="btn mt-2 mr-1" (click)="showQuestionYN();"><i
             class="icon-question-circle mr-1"></i>Question YN
     </button>
+    <button type="button" class="btn mt-2 mr-1" (click)="showCustomized();">Customized
```

Comment:
Non necessary ;

Reply:
true, I will fix also the other buttons click functions